src/chipjacks.cpp

```diff
@@ -113,14 +114,14 @@ void initialize_chipjack_and_memory_for_character(struct char_data *ch) {
   }
 }
 
-void set_skill_from_chip(struct char_data *ch, struct obj_data *chip, bool send_message, int skillwire_rating) {
-  if (!ch || !chip) {
-    mudlog_vfprintf(ch, LOG_SYSLOG, "SYSERR: Invalid parameters to set_skill_from_chip(%s, %s, %s, %d): null item!", GET_CHAR_NAME(ch), GET_OBJ_NAME(chip), send_message ? "TRUE" : "FALSE", skillwire_rating);
+void set_skill_from_chip(struct char_data *ch, struct matrix_file *skillsoft_file, bool send_message, int skillwire_rating) {
+  if (!ch || !skillsoft_file) {
+    mudlog_vfprintf(ch, LOG_SYSLOG, "SYSERR: Invalid parameters to set_skill_from_chip(%s, %s, %s, %d): null item!", GET_CHAR_NAME(ch), skillsoft_file->name, send_message ? "TRUE" : "FALSE", skillwire_rating);
```

Comment:
`skillsoft_file` can be null here, so `skillsoft_file ? skillsoft_file->name : "<null>"`